helmfile.d/charts/prometheus-alerts/templates/alerts/kubernetes-system-kubelet.yaml

```
@@ -176,24 +176,44 @@ spec:
 {{- if .Values.defaultRules.additionalRuleLabels }}
 {{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
 {{- end }}
-    - alert: LessKubeletsThanNodes
+    - alert: LessKubeletsThanAutoscalableNodes
```

What do you think of using "Autoscaled" instead of "Autoscalable"?

```suggestion
    - alert: LessKubeletsThanAutoscaledNodes
```
...

---

Was expecting some suggestions on the alert names :) 
I can change that for both alerts.